cyences_app_for_splunk/default/macros.conf

```diff
@@ -113,6 +113,27 @@ iseval = 0
 definition = `cs_tenable` sourcetype="tenable:io:vuln"
 iseval = 0
 
+#Ransomware
+[cs_system_processes_run_from_unexpected_locations_internal_filter]
+definition = search NOT process_path IN ("C:\\Program Files\\Common Files\\microsoft shared\\MSInfo\\msinfo32.exe","C:\\Users\\*\\AppData\\Local\\Temp\\*\\DismHost.exe","C:\\Users\\*\\AppData\\Local\\Microsoft\\OneDrive\\Update\\OneDriveSetup.exe","C:\\Users\\*\\AppData\\Local\\Microsoft\\OneDrive\\StandaloneUpdater\\OneDriveSetup.exe","C:\\Windows\\regedit.exe","C:\\Windows\\hh.exe","C:\\Users\\*\\AppData\\Local\\Temp\\WindowsUpdateLog\\tracerpt.exe")
```

Comment:
I think we should also check:
- C:\Program Files\Microsoft RDInfra\StackSxS\1.0.2103.26006\rdpshell.exe
- C:\Program Files\Microsoft RDInfra\StackSxS\1.0.2103.26006\rdpinit.exe

Reply:
I would like to discuss and will add it.